package.json

```diff
@@ -7,20 +7,22 @@
     "mysql": "^2.11.1"
   },
   "devDependencies": {
-    "babel": "^6.1.18",
+    "babel": "^6.23.0",
+    "babel-cli": "^6.24.1",
```

Comment:
hmm I think we can just use a global babel-cli for compiling... I just added this to have a npm command `npm run build` for compiling the library.